NitroxModel/Discovery/Models/GameInstallation.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace NitroxModel.Discovery.Models;
+
+public readonly record struct GameInstallation(
```

Comment:
I think this would be better as class. This struct only has reference types so it has no benefits for cache locality (to be a struct).